Packs/DeprecatedContent/pack_metadata.json

```diff
@@ -1,22 +1,18 @@
 {
-    "name": "Deprecated Content",
-    "description": "Deprecated Cortex XSOAR content pack.",
-    "support": "xsoar",
-    "currentVersion": "1.0.0",
-    "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex",
-    "email": "",
-    "categories": [
-        "Utilities"
-    ],
-    "tags": [
-        "Deprecated Content"
-    ],
-    "created": "2020-05-13T10:05:40Z",
-    "updated": "2020-05-13T10:05:40Z",
-    "beta": false,
-    "deprecated": false,
-    "useCases": [],
-    "keywords": [],
-    "dependencies": {}
+  "name": "Deprecated Content",
+  "description": "Deprecated Cortex XSOAR content pack.",
+  "support": "xsoar",
+  "currentVersion": "1.0.0",
+  "author": "Cortex XSOAR",
+  "url": "https://www.paloaltonetworks.com/cortex",
+  "email": "",
+  "created": "2020-05-13T10:05:40Z",
+  "categories": [
+    "Utilities"
+  ],
+  "tags": [
+    "Deprecated Content"
+  ],
+  "useCases": [],
+  "keywords": []
```

Comment:
shouldn't this one have `"deprecated": true`?

Reply:
In theory yes but server doesn't support this fields. That is why I removed it.